daemon/build.go

```diff
@@ -75,15 +76,24 @@ func (rl *releaseableLayer) Release() error {
 	if rl.released {
 		return nil
 	}
+	if err := rl.releaseRWLayer(); err != nil {
+		return err
+	}
+	if err := rl.releaseROLayer(); err != nil {
```

Comment:
Actually, never mind... that Get is a layerstore Get, which just grabs a reference without mounting.  Not the same as graphdriver Get, which actually mounts.  So I'll just adjust the error handling, not worry about roLayer being Put().